.github/workflows/tests.yml

```diff
@@ -36,18 +36,30 @@ jobs:
         run: |
           npx zx scripts/org-setup.mjs
 
-      - name: Run browser tests
+      - name: Run browser tests with 3 attempts
+        uses: nick-fields/retry@943e742917ac94714d2f408a0e8320f2d1fcafcd
+        with:
+          max_attempts: 3
+          timeout_minutes: 30
+          retry_wait_seconds: 60
+          command: |
+            SF_USERNAME=$(sf org display user --target-org jsforce-test-org --json | jq -r '.result.username') \
+            SF_PASSWORD=$(sf org display user --target-org jsforce-test-org --json | jq -r '.result.password') \
+            SF_LOGIN_URL=$(sf org display user --target-org jsforce-test-org --json | jq -r '.result.instanceUrl') \
+            DISPLAY=:99 CHROME_BIN=$(which chrome) \
+            npm run test:browser-ci
+          new_command_on_retry: |
+            SF_USERNAME=$(sf org display user --target-org jsforce-test-org --json | jq -r '.result.username') \
+            SF_PASSWORD=$(sf org display user --target-org jsforce-test-org --json | jq -r '.result.password') \
+            SF_LOGIN_URL=$(sf org display user --target-org jsforce-test-org --json | jq -r '.result.instanceUrl') \
+            DISPLAY=:99 CHROME_BIN=$(which chrome) \
+            npm run test:browser-ci:retry
```

Comment:
run the retry command if there was a failure in the first `npm run test:browser-ci` run
https://github.com/nick-fields/retry#run-different-command-after-first-failure